### PR TITLE
[ID] Add Feature Requests article

### PR DESCRIPTION
**Pull request by daud-ahn**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

I'm not sure about the obtaining stars with osu!supporter, is this still true because I never heard of it.

### CODE REVIEW

**Comment by TicClick** [comment]
> I'm not sure about the obtaining stars with osu!supporter, is this still true because I never heard of it.

it is: 1 month of supporter I just bought gave me 2 votes (918 -> 920)

File: wiki/Community/Forum/Feature_Requests/en.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ stub: true
 
 # Feature Requests
 
-[**Feature Requests**](https://osu.ppy.sh/community/forums/4) is a forum created for suggestions regarding osu!. Users can promote feature requests and increase their priority by [spending stars](/wiki/Modding/Star_priority) obtained with [osu!support](/wiki/osu!supporter).<!-- TODO: part of Modding/Star_priority belongs here -->
+[**Feature Requests**](https://osu.ppy.sh/community/forums/4) is a forum created for suggestions regarding osu!. Users can promote feature requests and increase their priority by [spending stars](/wiki/Modding/Star_priority) obtained with [osu!supporter](/wiki/osu!supporter).<!-- TODO: part of Modding/Star_priority belongs here -->
```
Comment (TPGPL) [inline wiki/Community/Forum/Feature_Requests/en.md:7]:
```suggestion
**[Feature Requests](https://osu.ppy.sh/community/forums/4)** is a forum created for suggestions regarding osu!. Users can promote feature requests and increase their priority by [spending stars](/wiki/Modding/Star_priority) obtained with [osu!supporter](/wiki/osu!supporter).<!-- TODO: part of Modding/Star_priority belongs here -->
```

File: wiki/Community/Forum/Feature_Requests/id.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Feature Requests
+
+[**Feature Requests**](https://osu.ppy.sh/community/forums/4) adalah forum yang dibuat untuk memberikan usulan terkait osu!. Pengguna dapat mempromosikan permintaan fitur dan meningkatkan prioritasnya dengan menggunakan [star](/wiki/Modding/Star_priority) yang diperoleh dengan [osu!supporter](/wiki/osu!supporter).
```
Comment (TPGPL) [inline wiki/Community/Forum/Feature_Requests/id.md:7]:
```suggestion
**[Feature Requests](https://osu.ppy.sh/community/forums/4)** adalah forum yang dibuat untuk memberikan usulan terkait osu!. Pengguna dapat mempromosikan permintaan fitur dan meningkatkan prioritasnya dengan menggunakan [star](/wiki/Modding/Star_priority) yang diperoleh dengan [osu!supporter](/wiki/osu!supporter).
```

**Review by TPGPL — DISMISSED** [review]

**Review by Nivalyx — DISMISSED** [review]
looks okay on the Indonesian side

**Review by cl8n — APPROVED** [review]

**Pull request merged**